build.sbt

```diff
@@ -20,7 +20,8 @@ val commonSettings = Seq(
     "com.github.ben-manes.caffeine" % "caffeine" % "2.3.5",
     "com.google.code.findbugs" % "jsr305" % "3.0.1", // required to prevent Caffeine causing compile to fail given -Xfatal-warnings flag.
     "com.amazonaws" % "amazon-kinesis-client" % "1.6.2",
-    "org.scalatest" %% "scalatest" % "2.2.6" % Test
+    "org.scalatest" %% "scalatest" % "2.2.6" % Test,
```

Comment:
Small thing, but stylistically test dependencies should always be at the bottom of your dependency list.